src/main/java/com/vntu/marenko/ualearning/server/model/Test.java

```diff
@@ -17,6 +17,12 @@ public class Test {
     @Column(name = "name_test")
     private String nameTest;
 
+    @Column(name = "description_test")
+    private String descriptionTest;
```

Comment:
Please update DB schema file when updating entities to keep information synchronised